CHANGELOG.md

```diff
@@ -48,6 +48,8 @@
 
 * `annotate/celltypist`: Fix missing values in annotation column caused by index misalignment (PR #976).
 
+* `workflows/annotation/scgpt_annotation`, `workflows/annotation/scgpt_integration_knn` and `workflows/integrate/scgpt_leiden`: Parameterization of HVG flavor (PR #979).
```

Comment:
Can we list this in bug fixes but also add something a breaking change or a notice at the top of the changelog?